GATK/3.8-1-0-gf15c1c3ef/VariantFiltration.nf

```diff
@@ -0,0 +1,24 @@
+process VariantFiltration {
+    tag {"GATK VariantFiltration ${analysis_id}"}
+    label 'GATK_3_8_1_0_gf15c1c3ef'
+    label 'GATK_3_8_1_0_gf15c1c3ef_VariantFiltration'
+    container = 'quay.io/biocontainers/gatk:3.8--py27_1'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+    tuple val(analysis_id), file(vcf_file), file(vcf_idx_file)
+
+    output:
+    tuple val(analysis_id), file("${vcf_file.baseName}.filter.vcf"), file("${vcf_file.baseName}.filter.vcf.idx")
+
+    script:
+    """
+    java -Xmx${task.memory.toGiga()-4}G -jar ${params.gatk_path} -T SelectVariants --reference_sequence ${params.genome} -V $vcf_file --out ${vcf_file.baseName}.snp.vcf --selectTypeToExclude INDEL
```

Comment:
I feel it would be more modular to split this into several processes (Select, Filter etc.) and create a small subworkflow.

Reply:
I agree, but splitting this in multiple processes would create a lot of extra overhead as these processes run for only 2 to 5 minutes in my current workflow. What we could do is rename `process VariantFiltration` to something that indicates that it is a combined process? Like we also did with 'BWA-mapping'. Maybe, `process VariantFiltrationSnpIndel`. This allows us to create `process VariantFiltration` as well for a process which only runs  gatk VariantFiltration?

Reply:
Good idea.

Reply:
I have pushed an update.